src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/DependenciesViewModelFactory.cs

```diff
@@ -32,8 +32,9 @@ public IDependencyViewModel CreateTargetViewModel(ITargetedDependenciesSnapshot
 
         public IDependencyViewModel CreateRootViewModel(string providerType, bool hasUnresolvedDependency)
         {
-            var provider = GetProvider(providerType);
-            return provider.CreateRootDependencyNode().ToViewModel(hasUnresolvedDependency);
+            IProjectDependenciesSubTreeProvider provider = GetProvider(providerType);
+            IDependencyModel dependencyModel = provider.CreateRootDependencyNode();
+            return dependencyModel.ToViewModel(hasUnresolvedDependency);
```

Comment:
Because I set breakpoints debugging this - couldn't set conditional breakpoints, see the commit: https://github.com/dotnet/project-system/pull/2392/commits/fcdeac4a1c5a1b9e2208c4cbf6d5066a02eb1168.